src/main/resources/hudson/plugins/sshslaves/Messages_es.properties

```diff
@@ -23,32 +23,32 @@
 # {0} [SSH] Remote file system root {1} does not exist. Will try to create it...
 SSHLauncher.RemoteFSDoesNotExist={0} [SSH] Directorio raiz en el sistema remoto ({1}) no existe. Intentando crearlo ...
 # {0} [SSH] Authentication successful.
-SSHLauncher.AuthenticationSuccessful={0} [SSH] Autenticaci�n v�lida
+SSHLauncher.AuthenticationSuccessful={0} [SSH] Autenticaci�n v�lida
```

Comment:
i didn't corrupt it, works in my IDE (c). Can get better encoding in Netbeans. New key/entry deletion won't help since I would have to save the file anyway. Ping @recena who was going to take a look